tests/sanity_helpers.py

```diff
@@ -0,0 +1,128 @@
+import logging
+
+from ocs_ci.framework import config
+from ocs_ci.ocs import constants, node
+from ocs_ci.utility.utils import ceph_health_check
+from ocs_ci.ocs.cluster import CephCluster
+
+from tests import helpers
+
+
+logger = logging.getLogger(__name__)
+
+
+def health_check(nodes):
+    """
+    Perform Ceph and cluster health checks
+    """
+    assert node.wait_for_nodes_status(nodes), (
+        f"Not all nodes reached status {constants.NODE_READY}"
+    )
+    ceph_cluster = CephCluster()
+    assert ceph_health_check(
+        namespace=config.ENV_DATA['cluster_namespace']
+    )
+    ceph_cluster.cluster_health_check(timeout=60)
+
+
+def create_resources(resources, run_io=True):
+    """
+    Sanity validation - Create resources (FS and RBD) and run IO
+
+    Args:
+        resources (tuple): Lists of projects, secrets, pools,
+            storageclasses, pvcs and pods
+        run_io (bool): True for run IO, False otherwise
+
+    """
+    # Create resources and run IO for both FS and RBD
+    # Unpack resources
+    projects, secrets, pools, storageclasses, pvcs, pods = resources[:6]
+
+    # Project
+    projects.append(helpers.create_project())
+
+    # Secrets
+    secrets.append(helpers.create_secret(constants.CEPHBLOCKPOOL))
+    secrets.append(helpers.create_secret(constants.CEPHFILESYSTEM))
+
+    # Pools
+    pools.append(helpers.create_ceph_block_pool())
+    pools.append(helpers.get_cephfs_data_pool_name())
+
+    # Storageclasses
+    storageclasses.append(
+        helpers.create_storage_class(
+            interface_type=constants.CEPHBLOCKPOOL,
+            interface_name=pools[0].name,
+            secret_name=secrets[0].name
+        )
+    )
+    storageclasses.append(
+        helpers.create_storage_class(
+            interface_type=constants.CEPHFILESYSTEM,
+            interface_name=pools[1],
+            secret_name=secrets[1].name
+        )
+    )
+
+    # PVCs
+    pvcs.append(helpers.create_pvc(
+        sc_name=storageclasses[0].name, namespace=projects[0].namespace)
+    )
+    pvcs.append(helpers.create_pvc(
+        sc_name=storageclasses[1].name, namespace=projects[0].namespace)
+    )
+
+    # Pods
+    pods.append(
+        helpers.create_pod(
+            interface_type=constants.CEPHBLOCKPOOL, pvc_name=pvcs[0].name,
+            namespace=projects[0].namespace
+        )
+    )
+    pods.append(
+        helpers.create_pod(
+            interface_type=constants.CEPHFILESYSTEM, pvc_name=pvcs[1].name,
+            namespace=projects[0].namespace
+        )
+    )
+    if run_io:
+        # Run IO
+        for pod in pods:
+            pod.run_io('fs', '1G')
```

Comment:
Will there be a possibility where we'll want to try a different sized IO? I think it might be a good idea to receive it from the user and just set the default size to be 1G

Reply:
we have an issue around this - https://github.com/red-hat-storage/ocs-ci/issues/455